packages/ui/src/working-groups/components/CreateOpening/DurationAndProcess.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+import { CKEditor } from '@/common/components/CKEditor'
+import { InputComponent, InputNumber } from '@/common/components/forms'
+import { Row } from '@/common/components/Modal'
+import { OpeningDuration } from '@/common/components/OpeningDuration/OpeningDuration'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TextBig } from '@/common/components/typography'
+
+interface Props {
+  hasHiringTarget?: boolean
+}
+
+export const DurationAndProcess = ({ hasHiringTarget }: Props) => {
+  return (
+    <RowGapBlock gap={24}>
+      <Row>
+        <RowGapBlock gap={20}>
+          <TextBig bold value>
+            Opening Duration
+          </TextBig>
+          <OpeningDuration label="Expected length of the application period" />
+
+          {hasHiringTarget && (
+            <InputComponent label="Hiring Target" required inputSize="m" name="durationAndProcess.target" tight>
```

Comment:
To correctly bind the label to the input you should provide the id to the `InputComponent`:
```suggestion
            <InputComponent id="hiring-target" label="Hiring Target" required inputSize="m" name="durationAndProcess.target" tight>
```
(also it will make the test easier with `getByLabelText`)